course4/homework-string/string.cc

```diff
@@ -4,6 +4,129 @@ namespace course {
 
 string::string()
 {
+	mStr = std::make_unique<char[]>(1);
+	mStr[0] = NULL;
+	mStrSize = 0;
 }
 
-}
\ No newline at end of file
+string::string(size_t n)
```

Comment:
What's the content of the string after the constructor has been called? Why would anyone use it? Consider making it explicit.